west.yml

```diff
@@ -229,7 +229,7 @@ manifest:
       groups:
         - hal
     - name: hal_stm32
-      revision: 366c5c909fbcecf94f32411887132024f61b0bcd
+      revision: 077e943b60f95fab515ddfcbb41210b863b76951
```

Comment:
This commit should be removed, we already point on 54724a815d23b7f518fa88eb5fb7e991109e191b which integrate h7rs hal

Reply:
refer to https://github.com/zephyrproject-rtos/zephyr/pull/73643